test_scripts/Capabilities/PersistingHMICapabilities/common.lua

```diff
@@ -0,0 +1,397 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local actions = require("user_modules/sequences/actions")
+local utils = require("user_modules/utils")
+local events = require('events')
+local runner = require('user_modules/script_runner')
+local SDL = require('SDL')
+local hmi_values = require("user_modules/hmi_values")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+config.defaultProtocolVersion = 2
+
+--[[ Shared Functions ]]
+local m = {}
+m.Title = runner.Title
+m.Step = runner.Step
+m.preconditions = actions.preconditions
+m.postconditions = actions.postconditions
+m.getHMICapabilitiesFromFile = actions.sdl.getHMICapabilitiesFromFile
+m.getMobileSession = actions.mobile.getSession
+m.getHMIConnection = actions.hmi.getConnection
+m.activateApp = actions.app.activate
+m.start = actions.start
+m.setSDLIniParameter = actions.sdl.setSDLIniParameter
+m.getSDLIniParameter = actions.sdl.getSDLIniParameter
+m.createSession = actions.mobile.createSession
+m.getParams = actions.app.getParams
+m.setHMIId = actions.app.setHMIId
+m.getPreloadedPT = actions.sdl.getPreloadedPT
```

Comment:
@VjKlepikov I would propose to do not pass this function to scripts. It is used in other function of common module.
actions.sdl.getPreloadedPT()
actions.sdl.setPreloadedPT()
utils.cloneTable()